Makefile

```diff
@@ -673,8 +673,9 @@ ifeq "$(BOOTSTRAPPING_FLEXDLL)" "true"
 # The recipe for runtime/ocamlruns$(EXE) also produces runtime/primitives
 boot/ocamlrun$(EXE): runtime/ocamlruns$(EXE)
 
-$(foreach runtime, ocamlrun ocamlrund ocamlruni, \
-  $(eval runtime/$(runtime)$(EXE): | $(BYTE_BINDIR)/flexlink$(EXE)))
+$(foreach runtime, ocamlrun$(EXE) ocamlrund$(EXE) ocamlruni$(EXE) \
+                   libcamlrun_shared$(EXT_DLL) libasmrun_shared$(EXT_DLL), \
```

Comment:
Is there a reason to prefer `$(EXT_DLL)` over `.$(SO)`? both are used, but I think it would make sense to switch fully to `.$(SO)` if we're using `.$(A)` and `.$(O)` everywhere, too.